### ISSUE TITLE
Instance to Instance, Peer to Peer, both, or neither?

### ISSUE THREAD

**Issue by julianlambert** [organizer-network/organizer.network#40]
**tldr;** We need a roadmap, what are we making?

Prompted by reading issue #33 and #39 I had a question about what is on the #19 roadmap.

Are we interested in creating a peer to peer distributed social network? Or a instance to instance federated social network? Is there opportunity to utilize features of both?

I can imagine a distributed network where instances essentially act as peer data hosts and web portals. In this fantasy, access to the application would be on the user's device over some peer to peer protocol. If you were an advanced or interested user you could establish your instance per instructrions and help enable access, transferrance speed, and data permenance. However this would be a pivot from what currently is an unfederated instance based web application.

I can also imagine organizer.network aiming to be a well built federated social network, in which case we should study the networks already in that space (beyond mastodon). IF we are aiming for a federated social network, what makes us different from any other one, what is our mission?
